### ISSUE TITLE
Bean update 7.x-1.13 Causes Duplicate Headers

### ISSUE THREAD

**Issue by kristydv** [govCMS/GovCMS7#970]
GovCMS updated to the latest version of Bean (1.13) on 28 Feb 2019, however, this update has caused the following issue: [Bean update 7.x-1.13 Causes Duplicate Headers
](https://www.drupal.org/project/bean/issues/2996282) 

This was uncovered by actioning a request from a support client I manage on behalf of Salsa, who had made a change to a Bean layout, and after saving the block, the headers showed up duplicate.

The temporary workaround was to remove the title from the block and adding it in the panel pane. 

A patch has been supplied in this the Drupal issue linked above, can we apply this patch to the GovCMS7 distribution?

**Comment by Belhendo** [comment]
Hi @kristydv 

Thanks for your question. We will need that patch merged into a stable release before we can use it.

Please raise another issue if we can help you further, I'm going to close this one.
